### DISCUSSION TITLE
Parameter panel filter

### DISCUSSION THREAD

**Discussion by hazcoper** [foxglove/community#440] [Q&A] [answered]
Add the the ability to choose/filter the parameters that appear on the parameter panel, or at least the ability to search for parameters

**Comment by rgov** [comment]
There was a similar request I saw in Slack for searching through a 3D scene with many objects. Perhaps this needs to be considered across the UI.

**Comment by Scout22** [comment]
Is there any news on this ?

**Comment by edgarcamilocamacho** [comment]
Hello, any conclusion about this?

**Comment by defunctzombie** [comment, marked as answer]
We've added a filter to the parameter panel to filter the list of parameters. This is available on the web version now and will be in the next desktop release.
